noir-projects/aztec-nr/aztec/src/context/call_interfaces.nr

```diff
@@ -16,16 +16,35 @@ pub trait CallInterface<let N: u32> {
 }
 
 pub struct PrivateCallInterface<let N: u32, T> {
```

Comment:
As Nico [proposed](https://github.com/AztecProtocol/aztec-packages/pull/12038#discussion_r1958931867) in a PR down the stack - makes sense to have the args private and instead use the new method. All the changes in this file are a reflection of that.